.github/workflows/ci.yml

```diff
@@ -17,17 +17,13 @@ jobs:
 
     phpunit:
         runs-on: ubuntu-latest
-        strategy:
-            matrix:
-                php-versions: ['8.0', '8.1']
-                prefer-lowest: [ '', '--prefer-lowest' ]
         steps:
             - uses: actions/checkout@master
             - uses: shivammathur/setup-php@master
               with:
-                  php-version: ${{ matrix.php-versions }}
+                  php-version: '8.1'
             - name: Install dependencies
-              run: composer update -n --prefer-dist  ${{ matrix.prefer-lowest }}
```

Comment:
We might still want to keep the `prefer lowest` matrix, to make sure that we test the tool with the whole range of allowed dependencies. Not having a lock file and supporting a pretty wide range of dependencies makes this pretty important imo